admin/WEBDAV_CONFIG_IN

```diff
@@ -0,0 +1,10 @@
+address: 0.0.0.0
```

Comment:
I'd call it `webdav/config.in`. (Note: `webdav/` being a sub-dir)

Reply:
Actually, if we're not replacing any values in the config file, we may just call it `webdav/config`. The `.in` suffix is commonly used when you're keeping placeholders in a file that will be dynamically replaced by the test/build.

Reply:
do you mean `admin/webdav/config` or directly `webdav/config`?

Reply:
`admin/webdav/config` - it's arguably overhead for just one file, but I can already see tests for backends requiring multiple files on the horizon.